pytest_inmanta/core.py

```diff
@@ -35,9 +35,9 @@
 except DistributionNotFound:
     CORE_VERSION = None
 
-
+# Setting a project-wide pip index is only supported for iso7+
 SUPPORTS_PROJECT_PIP_INDEX: bool = (
-    CORE_VERSION is not None and CORE_VERSION >= version.Version("9.2.0.dev")
+    CORE_VERSION is not None and CORE_VERSION >= version.Version("10.0.0.dev")
```

Comment:
This is the right version number for iso7+ ?

Reply:
9.0.0.dev